src/zampy/datasets/cds_utils.py

```diff
@@ -152,18 +156,27 @@ def cds_request_land_cover(
     years_months = time_bounds_to_year_month(time_bounds)
     years = {year for (year, _) in years_months}
 
+    if spatial_bounds is not None:
+        area = [
+            spatial_bounds.north,
+            spatial_bounds.west,
+            spatial_bounds.south,
+            spatial_bounds.east,
+        ]
+
     for year in tqdm(years):
         if int(year) < 2016:
-            version = "v2.0.7cds"
+            version = "v2_0_7cds"
```

Comment:
That's a fun change in the cdsapi :roll_eyes: 
Hope they don't 'fix' that again later, breaking this.